recipes/volk/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class VolkConan(ConanFile):
+    name = "volk"
+    license = "MIT"
+    homepage = "https://github.com/zeux/volk"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "volk is a meta-loader for Vulkan. It allows you to \
+    dynamically load entrypoints required to use Vulkan without linking\
+     to vulkan-1.dll or statically linking Vulkan loader. Additionally,\
+      volk simplifies the use of Vulkan extensions by automatically\
+       loading all associated entrypoints. Finally, volk enables\
+        loading Vulkan entrypoints directly from the driver which \
+        can increase performance by skipping loader dispatch \
+        overhead."
+    topics = ("Vulkan", "loader", "extension", "entrypoint", "graphics")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+        def configure(self):
+            del self.settings.compiler.cppstd
+            del self.settings.compiler.libcxx
+
+        def configure(self):
+            del self.settings.compiler.cppstd
+            del self.settings.compiler.libcxx
+
+        def configure(self):
+            del self.settings.compiler.cppstd
+            del self.settings.compiler.libcxx
+
+    def requirements(self):
+        if self.version <= 20200318:
+            self.requires("vulkan-headers/1.2.135.0")
+        else:
+            self.requires("vulkan-headers/1.2.140.0")
```

Comment:
Does it matter what actual vulkan version a volk library is built against?
Can't it just use the latest? Consumers can always use another version in their own require section.

Reply:
I would think it doesn't matter as the volk library is a loader.
As a loader should be version indifferent, it should not matter what version it is built against.
Then again, the higher the vulkan version the volk loader is built against, the more it will support.

Reply:
makes sense.

Reply:
Vulkan-headers are not backward compatible. So it depends on which declarations are used in volk.